index.html

```diff
@@ -1,10 +1,521 @@
+
+        </div>
+        <!-- Trending news section ends   -->
+
+    </div>
+    <div class="clear hidden">A</div>
+
+    <footer>
+        <hr style="margin-bottom: 2px">
+        <img src="./image/NYT Logo.png" alt="logo" height="25">
+        <a href="#" class="small-text right">Go to Home Page <span style="font-size: 9px; vertical-align: middle">&Gt;</span></a>
+        <hr style="visibility:hidden">
+        <div class="footer">
+            <ul>
+                <li>NEWS</li>
+                <li>Home Page</li>
+                <li>U.S.</li>
+                <li>Politics</li>
+                <li>New York</li>
+                <li>Business</li>
+                <li>Tech</li>
+                <li>Science</li>
+                <li>Sports</li>
+                <li>Obituaries</li>
+                <li>Today's Paper</li>
+                <li>Corrections</li>
+            </ul>
+        </div>
+        <div class="footer">
+            <ul>
+                <li>OPINION</li>
+                <li>Today's Opinion</li>
+                <li>Op-Ed Columnists</li>
+                <li>Editorials</li>
+                <li>Op-Ed Contributors</li>
+                <li>Letters</li>
+                <li>Sunday Review</li>
+                <li>Video: Opinion</li>
+            </ul>
+        </div>
+        <div class="footer">
+            <ul>
+                <li>ARTS</li>
+                <li>Today's Arts</li>
+                <li>Art & Design</li>
+                <li>Books</li>
+                <li>Dance</li>
+                <li>Movies</li>
+                <li>Music</li>
+                <li>Pop Culture</li>
+                <li>Television</li>
+                <li>Theater</li>
+                <li>Video: Arts</li>
+            </ul>
+        </div>
+        <div class="footer">
+            <ul>
+                <li>LIVING</li>
+                <li>Automobiles</li>
+                <li>Crossword</li>
+                <li>Food</li>
+                <li>Education</li>
+                <li>Style</li>
+                <li>Health</li>
+                <li>Jobs</li>
+                <li>Magazine</li>
+                <li>Real Estate</li>
+                <li>T Magazine</li>
+                <li>Travel</li>
+                <li>Weddings</li>
+            </ul>
+        </div>
+        <div class="footer">
+            <ul>
+                <li>MORE</li>
+                <li>Reader Center</li>
+                <li>Wirecutter</li>
+                <li>Live Events</li>
+                <li>The Learning Network</li>
+                <li>Tools & Services</li>
+                <li>N.Y.C. Events Guide</li>
+                <li>Multimedia</li>
+                <li>Photography</li>
+                <li>Video</li>
+                <li>NYT Store</li>
+                <li>Times Journeys</li>
+                <li>Manage My Account</li>
+            </ul>
+        </div>
+
+
+        <div class="footer">
```

Comment:
You are missing the NY Times image, the font should be sans-serif and the icons are repeated